tests/run-macros/i6679/Macro_1.scala

```diff
@@ -0,0 +1,14 @@
+import scala.quoted._
+
+def makeMatch[A: Type](head : Expr[A]) given (qctx : QuoteContext) : Expr[Unit] = {
+  import qctx.tasty._
+
+  val sacrifice = '{ $head match { case _ => ??? } }
+  sacrifice.unseal
+
+  '{ println("Ok") }
```

Comment:
put `sacrifice` inside the quote?

Reply:
We already show that it can be pickled and unpickled. Putting it in a quote would not touch that code further.